### PR TITLE
Update design

### PR DESCRIPTION
**Pull request by kr8n3r**
Non `.service` things [should not look like gov.uk ](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk#if-your-service-isnt-on-govuk)

This redesign applies custom design, based on the mascot colour.

Uses css grid for dynamic layout based on number of columns (say "rejected")

localhost_8080__

Sticky headings
Screenshot 2021-07-14 at 08 13 59

## How to test

- `cf env rubbernecker` from production, and steal the creds
- Export stolen certs into an environment
- `go run .`

### CODE REVIEW

**Review by paroxp — APPROVED** [review]
Must have been cooking this for a while.

**Comment by kr8n3r** [comment]
> Must have been cooking this for a while.

since this morning

File: dist/css/application.css [hunk 1]
```diff
@@ -1,37 +1,168 @@
...
 }
 
-.text-center {
+h3 {
+  font-size: 1.2rem;
+}
+
+.heading {
+  margin: 0;
+  padding: 0;
```
Comment (paroxp) [inline dist/css/application.css:42]:
```suggestion
  box-shadow: 0px 10px 10px -5px #ddd;
  margin: 0;
  margin-bottom: .5em;
  padding: .5em;
```

What do you think?

Little eye candy, making some space between the edges of the div (specially the top one), as well as making it simpler to see the heading when sticky?

File: dist/css/application.css [hunk 2]
```diff
@@ -1,37 +1,168 @@
...
+.filter--active {
+  text-decoration-color: #f78932;
+}
+
 .card {
   background: #fAfAfA;
   min-height: 100px;
-  width: 100%;
   margin-bottom: 5px;
   margin-right: 5px;
```
Comment (paroxp) [inline dist/css/application.css:133]:
```suggestion
```

File: dist/css/application.css (hunk 1 above)
Comment (kr8n3r) [reply dist/css/application.css:42]: sounds good, will check out the change tmrw

File: dist/css/application.css (hunk 2 above)
Comment (kr8n3r) [reply dist/css/application.css]: tweaked a bit 
Screenshot 2021-07-14 at 08 05 08

**Pull request merged**